labellab_mobile/lib/routing/route_handler.dart

```diff
@@ -46,7 +47,8 @@ import 'package:labellab_mobile/screen/train/model_train_screen.dart';
 import 'package:provider/provider.dart';
 
 import '../screen/issue/add_edit/add_edit_issue_screen.dart';
-import '../screen/issue/issue_activity_screen.dart';
+import '../screen/issue/details/issue_details_screen.dart';
+import '../screen/issue/issue_activity/issue_activity_screen.dart';
```

Comment:
In case you didn't know, absolute import means something like this -

```
import 'package:labellab_mobile/screen/issue/issue_activity/issue_activity_screen.dart';

Please fix these
````